README.md

```diff
@@ -65,25 +78,16 @@ Create the child application (see [docs](/docs/child-applications.md) for more d
 document.getElementById('main-content').textContent += "App1 is loaded.";
 
 // apps/app1.js
-export function bootstrap() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent += "App1 is bootstrapped.";
-    resolve();
-  });
+export async function bootstrap() {
+  document.getElementById('main-content').textContent += "App1 is bootstrapped.";
 }
 
-export function mount() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent += "App1 is mounted!";
-    resolve();
-  });
+export async function mount() {
+  document.getElementById('main-content').textContent += "App1 is mounted!";
 }
 
-export function unmount() {
-  return new Promise((resolve, reject) => {
-    document.getElementById('main-content').textContent = "";
-    resolve();
-  });
+export async function unmount() {
+  document.getElementById('main-content').textContent = "";
 }
```

Comment:
I :heart: the async functions, but would this make some people a little nervous who aren't familiar with the syntax?